aea/mail/base.py

```diff
@@ -209,7 +209,7 @@ def _get_public_ids_from_uri(
             )
         return (skill_id, connection_id)
 
-    def __str__(self):
+    def __str__(self):  # pragma: nocover
```

Comment:
This should have a test. These methods can easily break.

Reply:
done